### ISSUE TITLE
Fine tuning Loss function

### ISSUE THREAD

**Issue by evardhen** [evardhen/VLM-Grocery-Fine-Tuning#2]
Think about alternative loss functions than the standard one, for better estimate of products counts
